src/front_input.c

```diff
@@ -1289,6 +1289,27 @@ short get_creature_control_action_inputs(void)
             }
             message_add(CrInst, get_string(StrID));
         }
+        if (is_key_pressed(KC_SEMICOLON,KMod_NONE))
```

Comment:
Instead of a hardcoded SEMICOLON, could you move it to the 'POSSESSION' key?

Reply:
Which key is that? I can't find it in front_input.h.

Reply:
Never mind; I think I've found it.

Reply:
You did, great, thanks.